crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -171,3 +206,76 @@ pub struct Capsule {
 }
 impl super::Primitive2d for Capsule {}
 impl Primitive3d for Capsule {}
+
+impl Capsule {
+    /// Create a new `Capsule` from a radius and half-length
+    pub fn new(radius: f32, half_length: f32) -> Self {
+        Self {
+            radius,
+            half_length,
+        }
+    }
```

Comment:
Other constructors take full extents. The length could also be explained more, something like this?
```suggestion
    /// Create a new `Capsule` from a radius and the full length of the line segment
    /// connecting the hemispheres
    pub fn new(radius: f32, length: f32) -> Self {
        Self {
            radius,
            half_length: length / 2.0,
        }
    }
```